SimpleServlet/src/main/java/com/skankhunt220/repository/UserRepository.java

```diff
@@ -0,0 +1,54 @@
+package main.java.com.skankhunt220.repository;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import main.java.com.skankhunt220.entities.User;
+
+public class UserRepository {
+	private MongoDatabase database;
+	private MongoCollection<Document> collection;
+
+	public void create(User user) {
+		try (MongoClient mc = new MongoClient("localhost", 27017)) {
+			database = mc.getDatabase("UserDB");
+			collection = database.getCollection("users");
+
+			collection.insertOne(new Document("_id", new ObjectId()).append("firstName", user.getFirstName())
+					.append("middleName", user.getMiddleName()).append("lastName", user.getLastName()));
+		}
+	}
+
+	public Document read(String userId) {
+		try (MongoClient mc = new MongoClient("localhost", 27017)) {
+			database = mc.getDatabase("UserDB");
+			collection = database.getCollection("users");
+			Document myDoc = collection.find(eq("_id", new ObjectId(userId))).first();
```

Comment:
This variable absolutely unnecessary, because it using only once, so rewrite like this:
`return collection.find(eq("_id", new ObjectId(userId))).first();`